src/logic.js

```diff
@@ -392,10 +435,9 @@ define([
                 }
             }, _this = this;
 
-            ret.preload = _.chain(this.all)
-                .filter(function(ref) {
-                    return isCaseReference(ref.path);
-                })
+            ret.preload = _.chain(this.reverse[CASE_REF_ID] || {})
+                .values()
+                .flatten(true)
                 .map(function(ref) {
                     var info = ref.path.split('/'),
                         prop = info[2];
```

Comment:
Eek. looks like we missed that when we changed the syntax. I'll change it in a different pr